src/domain/profile/events/ProfileEventsList.tsx

```diff
@@ -23,21 +24,23 @@ import Icon from '../../../common/components/icon/Icon';
 
 interface ProfileEventsListProps {
   availableEvents: AvailableEventsTypes | null;
+  child: ChildByIdResponse;
   enrolments: EnrolmentsTypes;
   pastEvents: PastEventsTypes | null;
 }
 
 const EVENT_DURATION_MINUTES = 30; // TODO: huh?
 const ProfileEventsList: FunctionComponent<ProfileEventsListProps> = ({
   availableEvents,
+  child,
```

Comment:
Do you think we can - or if we should - circumvent passing the whole child data for this component? It seems to only need the `id`, or, semantically, the url base.

Reply:
I think we can! And I don't know if we should, I considered just passing the id and decided to pass the whole shebang thinking it could be convenient in the future. Would it help us prevent bugs or improve performance to only send the id?

Reply:
I would say it is more likely to prevent bugs (and keep the components' responsibilities as small as possible) than to improve performance.

Reply:
Makes sense. I'll update the PR with this in mind.